components/MainDonationForm.js

```diff
@@ -14,10 +14,18 @@ export default function MainDonationForm() {
     }
 
     if (succeed) {
+      const causes = event.target.cause;
+      const validCause = [];
```

Comment:
```suggestion
      const selectedCauseList = [];
```

Reply:
I really like how you implemented this. Just a tiny improvement: I renamed `validCause` to `selectedCauseList` to be more explicit.